<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ELP :: Proj. flyback feedback loop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icone.png</normaloff>img/icone.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>661</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Proj. Trafo</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>241</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Configuração do transformador</string>
     </property>
     <widget class="QLCDNumber" name="disp_nep">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="disp_nes">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="disp_sc">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="disp_lmp">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="disp_lms">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N° de espiras no primário</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N° de espiras no secundário</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seção dos condutores</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>L magnetizante do pri. (uH)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>L magnetizante do sec. (uH)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_pi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Processar informações</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ae*Aw calculado</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="disp_calc">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="disp_dent">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dimetro do entreferro (mm)</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>341</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Modelo Thornton selecionado</string>
     </property>
     <widget class="QLabel" name="label_model">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagem do modelo selecionado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>label_model</zorder>
     <zorder>disp_calc</zorder>
     <zorder>label_9</zorder>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>341</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Modelos de transformadores Thornton</string>
     </property>
     <widget class="QTableWidget" name="tabela_model">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>320</width>
        <height>192</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_sa">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modelo indicado</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_cm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Carrega modelos</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Definições iniciais</string>
     </property>
     <widget class="QTableWidget" name="tabela_def">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_calc">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>250</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular Ae*Aw</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Observação:</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
